system.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// StopEtcd stops etcd on a specific host
+func StopEtcd(node TestbedNode) error {
+	log.Infof("Stopping etcd")
+	return node.RunCommand("pkill etcd && rm -rf /tmp/etcd")
+}
+
+// StartEtcd starts etcd on a specific host.
+func StartEtcd(node TestbedNode) error {
+	log.Infof("Starting etcd")
+
+	_, err := node.RunCommandBackground("nohup etcd -data-dir /tmp/etcd </dev/null &>/dev/null &")
```

Comment:
I can add logging to /tmp. Work for you?

On 11 Oct 2015, at 9:38, Vipin Jain wrote:

>> +	"time"
>> +
>> +	log "github.com/Sirupsen/logrus"
>> +)
>> +
>> +// StopEtcd stops etcd on a specific host
>> +func StopEtcd(node TestbedNode) error {
>> +	log.Infof("Stopping etcd")
>> +	return node.RunCommand("pkill etcd && rm -rf /tmp/etcd")
>> +}
>> +
>> +// StartEtcd starts etcd on a specific host.
>> +func StartEtcd(node TestbedNode) error {
>> +	log.Infof("Starting etcd")
>> +
>> +	_, err := node.RunCommandBackground("nohup etcd -data-dir /tmp/etcd 
>> </dev/null &>/dev/null &")
>
> where are etcd logs kept? This is to help debug things? Do we need 
> similar things for consul?
>
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/systemtests-utils/pull/3/files#r41712580